toolkit/resources/manifests/package/pkggen_core_x86_64.txt

```diff
@@ -155,8 +155,8 @@ npth-1.6-3.cm1.x86_64.rpm
 pinentry-1.1.0-3.cm1.x86_64.rpm
 gnupg2-2.2.20-3.cm1.x86_64.rpm
 gpgme-1.13.1-5.cm1.x86_64.rpm
-mariner-repos-1.0-11.cm1.noarch.rpm
-mariner-repos-preview-1.0-11.cm1.noarch.rpm
+mariner-repos-1.0-12.cm1.noarch.rpm
```

Comment:
You'll also need to update `toolkit/resources/manifests/package/update_manifests.sh` to exclude `mariner-repos-extra` from the worker chroot since it's wildcard include `mariner-repos-*`.